utils.sh

```diff
@@ -79,7 +79,7 @@ function provide-parity-tools {
 
     export PATH=~/.cargo/bin:$PATH
 
-    cargo install --git https://github.com/paritytech/cargo-contract.git cargo-contract --force
+    cargo install cargo-contract
```

Comment:
@ascjones is `cargo-contract` published often enough to be installed locally like this?

Reply:
Robin says we should get stable version from crates.io. I therefore argue to just do a plain install without force which would only re-install when a new version is available.

Reply:
You mean on Riot? That was just his answer to my question if the readme in cargo-contract is outdated. We don't want the latest stable versions here, but the latest versions to catch integration bugs early on.

Reply:
+1 @Stefie

Reply:
If you insist on installing from master then I would prefer to put it into the ci script. It is excruciating during development to re-install the same version of that tool all over again.

Reply:
Let's use the stable crates.io release. Integration tests for `cargo-contract/master` can live in that repo.